emmaa/analyze_tests_results.py

```diff
@@ -446,9 +451,14 @@ def make_model_delta(self):
             self.json_stats['model_delta'] = {
                 'statements_hashes_delta': {'added': [], 'removed': []}}
         else:
+            stmts_delta = self.latest_round.find_delta_hashes(
+                self.previous_round, 'statements')
             self.json_stats['model_delta'] = {
-                'statements_hashes_delta': self.latest_round.find_delta_hashes(
-                    self.previous_round, 'statements')}
+                'statements_hashes_delta': stmts_delta}
+            msg = _make_twitter_msg(self.model_name, 'stmts', stmts_delta,
+                                    self.latest_round.date_str[:10])
+            if msg:
+                logger.info(msg)
```

Comment:
This is optional, as it's only for logging the twitter messages during Batch jobs